docs/turn-howto.md

```diff
@@ -234,6 +261,17 @@ Here are a few things to try:
    Try removing any AAAA records for your TURN server, so that it is only
    reachable over IPv4.
 
+ * If your TURN server is behind a NAT:
+
+    * double-check that your NAT gateway is correctly forwarding all TURN
+      ports to the NAT-internal address of your TURN server. If advertising
```

Comment:
should we expand on "all TURN ports"? in my experience, people remember to forward 3478 and 5349, but forget about the UDP relay ports (49152-65535).

Reply:
Listed the ports in the latest commit.